docs/content/direct/user-guide.md

```diff
@@ -0,0 +1,75 @@
+# User Guide
+
+Status of this document: it is the barest of a start. Much more needs to be written.
+
+This document is for users of a release. Examples of using the latest stable release are in [the examples document](examples.md). This document adds information not conveyed in the examples.
+
+## Using a pre-existing cluster as the hosting cluster
+
+See [Using an existing hosting cluster](./hosting-cluster.md)
+
+## When everything is not on the same machine
```

Comment:
Do you mean same machine or same cluster ? 
What do you mean when you say "Everything"  ? Is it just spaces? what about having KF running on one cluster and KS on another?  How about the OTP plugin can it run on a different cluster? 
I think we should drop that section completely at this point and put Under Construction

Reply:
I mean same machine. By "everything" I mean the components in the KubeStellar architecture, everything in scope for this document.

Already in the examples based on `kind` clusters, there are KubeStellar components in different clusters: there is a hosting cluster, spaces created by KubeFlex running in that hosting cluster, and two WECs that are outside of the hosting cluster. This section is not about changing the architecture (e.g., the fact that the transport controller runs in the hosting cluster), it is about choices for using the existing architecture.

The existing architecture does not say that all the WECs have to run on the same machine as the hosting cluster. However, at the current level of development, it is possible to beyond a one-machine system only if the hosting cluster is OpenShift. This is worth explaining, even though it is disappointing. So I do not think this material should be deleted.

Reply:
I'm not sure all components can be indeed separated to different clusters (not by design but due to bugs) , but will not stop this PR due to this section as it's not really related to it.